components/Card.vue

```diff
@@ -123,16 +131,19 @@ export default Vue.extend({
           case 'Audio':
           case 'Folder':
           case 'MusicAlbum':
+          case 'MusicArtist':
+          case 'MusicGenre':
```

Comment:
Is there any chance the cards scale to match the height of the sliders now? If that were the case, we could potentially mix items within a single slider for stuff like the search page or the playlist at some point.

Reply:
Technically, nothing prevents us from doing so (Though the cards have different heights, but the slider will take whatever height is the largest).

I don't think making all the cards fit into a 1:1.5 card would look good, though.